.github/workflows/weekly-serverless.yml

```diff
@@ -11,12 +11,22 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       deployment_name: ${{ steps.set_deployment_name.outputs.date-name }}
+      stack_version: ${{ steps.set_deployment_name.outputs.stack-version }}
     steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Hermit Environment
+        uses: ./.github/actions/hermit
+
       - name: Set deployment name
         id: set_deployment_name
+        env:
+          STACK_VERSION: ${{ env.ELK_VERSION }}
```

Comment:
is this set by hermit? is this why we added that step?

Reply:
Exactly, instead of using a hardcoded version, we can retrieve it from the Hermit environment variable.